Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6.8
+
+WORKDIR /root/src/clever
```

Comment:
Change this to /usr/src/kruize

Reply:
Updated the path. Missed to update it earlier.